client/src/pages/Courses/editables/IsPlaceholderField.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from "react";
+import { Flex, Text, Checkbox, Tag } from "@chakra-ui/core";
+import { colors as c } from "../../../colors";
+
+const IsPlaceholderField = ({ value, isEditing, onChange }) => {
+    const [isPlaceholder, setIsPlaceholder] = useState([]);
+    useEffect(() => {
+        setIsPlaceholder(value);
+    }, [value]);
+
+    const updateIsPlaceholder = (c) => {
+        setIsPlaceholder(c);
+        onChange("placeholder", c);
+    };
+
+    return (
+        <Flex>
+            {!isEditing ? (
+                value && (
+                    <Flex justify="center">
+                        <Tag height="10px" mt="20px" size="md" rounded="full" variant="solid" variantColor="teal" mr={1}>
+                            Placeholder
+                        </Tag>
+                    </Flex>
+                )
+            ) : (
+                <Flex direction="column" alignItems="center">
+                    <Text textAlign="center" color={c.darkBlue} fontWeight="bold">
+                        Placeholder Course
```

Comment:
Can you please add a tooltip that the user can hover to understand what a placeholder course really means? So the icon with the (?) that might say:

`A placeholder course can be added multiple times within the same plan.`